teapots/common/ndk_helper/JNIHelper.h

```diff
@@ -151,23 +235,31 @@ class JNIHelper {
   std::string GetExternalFilesDir();
```

Comment:
if you talking about make the function to be const as:
 std::string GetExternalFilesDir() const;
it will not compile as all functions you called inside this function has to have "const" character ( like AttachCurrentThread() etc )...